src/app/component/CookiePopup/CookiePopup.component.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { PureComponent } from 'react';
+import BrowserDatabase from 'Util/BrowserDatabase';
+import Link from 'Component/Link';
+import './CookiePopup.style';
+
+export const COOKIE_POPUP = 'cookie_popup';
+
+class CookiePopup extends PureComponent {
+	constructor() {
+		super();
+
+		this.state = {
+			isAccepted: BrowserDatabase.getItem(COOKIE_POPUP) || false
+		};
+
+		this.acceptCookies = this.acceptCookies.bind(this);
```

Comment:
Prefer declaring this on class itself, no need for constructor for both state and acceptCookies function.